DOCS-14648 CSFLE template clarification (#757)
mongodb/docs-ecosystem: source/includes/substitute-placeholders.rst

> .. note::

Reviewer: Suggestion: Maybe add a heading? I don't think this note is very scannable.

```suggestion
.. note:: Placeholder Text
```

>     You must substitute all values in quotes with angle brackets with

Reviewer: ```suggestion
    You must substitute all text in quotes and angle brackets with
```
Suggestion: "In quotes with angle brackets" sounded a bit off to me. Is this a value in quotes with angle brackets: "hello <>"? Also what is a value? Are all values text in strings?

>     your KMS configuration values. 

Reviewer: Suggestion: Maybe add an example replacement? Definitely don't have to use tabs this is just the first thing that came to mind. 

```suggestion
    your KMS configuration values. 
    
    For example, if your username is "MongoUser", you should replace the text in the following code snippet as follows:
    
    .. tab :: unreplaced
     
       .. code-block:: java
    
          "<your username>"

      .. tab :: replaced 
     
       .. code-block:: java
    
          "MongoUser"    
    
```
